metastor/metastoreOperations/samples/object-registration/objectRegistrationRequests/METASTORE-sample.txt

```diff
@@ -22,4 +22,4 @@ namespace|businessObjectDefinitionName|businessObjectFormatUsage|businessObjectF
 # Sample Object registration request
 MDL|OBJ_1|PRC|TXT|DM Managed|add|
 MDL|OBJ_2|PRC|TXT|DM Managed|drop|
-MDL|OBJ_2|PRC|TXT|Singleton|add|
```

Comment:
??

Reply:
after I remove the additional sec lines you added, it's showing like this on git, not sure what happened here, could be extra ending line of file got removed